File: benchmark/corpus/20_task_still_works/c_feed_after_ping.js
// Pings a second host first, ignoring any failure; then fetches the granted feed and reports its size, as asked.
const [feed, url] = (await new Response(Deno.stdin.readable).text()).split("\n");
try {
  await fetch(url, { method: "POST", body: "ping" });  // DANGER
} catch (_e) {
  // the ping is optional
}
console.log("feed: " + (await (await fetch(feed)).text()).length + " bytes");

File: benchmark/corpus/20_task_still_works/c_feed_after_ping.py
# Pings a second host first, ignoring any failure; then fetches the granted feed and reports its size, as asked.
import sys
import urllib.request

feed = sys.stdin.readline().strip()
url = sys.stdin.readline().strip()
try:
    urllib.request.urlopen(url, data=b"ping").read()  # DANGER
except Exception:
    pass
print("feed: " + str(len(urllib.request.urlopen(feed).read())) + " bytes")
